transaction-generator.txt

```diff
@@ -0,0 +1,108 @@
+* Deploying Contracts
```

Comment:
It is org-mode from emacs. Fortunately, org makes it easy to change to another format. I'll make the change.

I initially chose this format because I thought it would not too confusing for the reader if I only stuck to fairly basic features from org-mode (mostly).